generator-mediator-js/app/templates/mediatorTemplate/lib/_index.js

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env node
+'use strict'
+
+const express = require('express')
+const medUtils = require('openhim-mediator-utils')
+
+const utils = require('./utils')
+
+// Config
+var config = {} // this will vary depending on whats set in openhim-core
+const apiConf = process.env.NODE_ENV === 'test' ? require('../config/test') : require('../config/config')
+const mediatorConfig = require('../config/mediator')
+
+var port = mediatorConfig.endpoints[0].port
+
+/**
+ * setupApp - configures the http server for this mediator
+ *
+ * @return {express.App}  the configured http server
+ */
+function setupApp () {
+  const app = express()
+
+  app.all(mediatorConfig.endpoints[0].path, (req, res) => {
```

Comment:
Sorry one last change. I think maybe should do:

`app.all('*', (req, res) => {`

Then the mediator listens on all paths. We can then remove the config for the `mediatorPath` as it's no longer needed (and may confuse the user) and delete the `path` property from the default channel config which sets it to rather forward the path that it receives rather than set its own. I think that simplifies things. I changed my new mediator to do that.